stuff/canonicalize_tags.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python3
+
+'''Ensures the usage of just canonical tags on quality nominations.
+
+This script reads all quality nominations suggestions contents
+and ensures that only canonical tags are used.
+'''
+
+import argparse
+import json
+import logging
+import os
+import unicodedata
+import re
+from typing import Mapping, Set
+
+import MySQLdb.constants.ER
+
+import cron.lib.db as db
+import cron.lib.logs as logs
+
+LOCALIZATIONS_ROOT = os.path.abspath(os.path.join(__file__,
+                                                  '..', '..',
+                                                  'frontend/www/js/omegaup'))
+LANGS = ['en', 'es', 'pt']
+
+
+def normalize_tag(tag: str) -> str:
+    '''Normalizes tags, similar to Tags::normalize() in PHP'''
+    # Remove empty spaces
+    tag = tag.strip()
+    # Remove accents
+    tag = unicodedata.normalize('NFD',
+                                tag).encode('ascii', 'ignore').decode("utf-8")
+    # Use '-' for splitting if necessary
+    tag = re.sub(r'[^a-z0-9]+', '-', tag.lower())
+    return tag
+
+
+def insert_new_tags(tags: Set[str],
+                    dbconn: MySQLdb.connections.Connection) -> None:
+    '''Inserts new problem tags inside Tags table on DB'''
+    logging.info('Inserting new Tags on database')
+    with dbconn.cursor() as cur:
+        cur.executemany('''INSERT IGNORE INTO `Tags`(`name`)
+                        VALUES (%s);''',
+                        [(tag,) for tag in tags])
+
+
+def get_inverse_mapping(
+        dbconn: MySQLdb.connections.Connection) -> Mapping[
+            str, Mapping[str, str]]:
+    '''Gets the inverse mapping for problem tags entries in lang files'''
+    logging.info('Getting tags mapping from lang files.')
+    new_tags = set()
+    inverse_mapping = {}
```

Comment:
```suggestion
    inverse_mapping: DefaultDict[Text, Dict[Text, Text]] = collections.defaultdict(dict)
```

así no necesitas L61/L67 y puedes hacer

```python
                    inverse_mapping[lang][normalize_tag(value)] = normalized_tag
```